projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -98,6 +109,29 @@ export default function getRecordInfo( postCount, postTypeBreakdown, lastIndexed
 	};
 }
 
+/**
+ * adds the appropriate labels the post type breakdown
+ *
+ * @param {Array} postTypeBreakdown - an array of the different post types with their counts
+ * @param {Array} postTypeLabels - an array of the different post types labels matched with their slugs
+ * @returns {object} updated postTypeBreakdown containing the post type slug, label, and count
+ */
+export function addLabelsToPostTypeBreakdown( postTypeBreakdown, postTypeLabels ) {
+	const postTypeBreakdownWithLabels = [];
+	for ( const postType of postTypeBreakdown ) {
```

Comment:
Rather than using nested `for`s, could we express this a bit more elegantly using `map`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Good examples in here:

https://egghead.io/lessons/javascript-use-map-instead-of-a-for-loop-to-iterate-over-an-array

Reply:
Ah, absolutely! I've used this previously for this sort of thing but somehow missed it this time. Good catch!